TASVideos/Pages/Shared/_UserFileInfo.cshtml

```diff
@@ -75,14 +75,16 @@
 		</div>
 	</cardbody>
 </div>
+
 <div condition="!string.IsNullOrWhiteSpace(Model.Annotations)" class="mb-2">
-	<card>
+	<div class="card border-primary-subtle border mb-2">
```

Comment:
card creates div class="card", it is better to keep it as card and simply add the other classes